src/pages/lend.tsx

```diff
@@ -14,19 +20,68 @@ export default function Lend() {
     }
   );
 
+  const displayBonds = useMemo(() => {
+    switch (bondFilter) {
+      case "current":
+        return nfts?.filter(nft => nft.status === "bonding");
+
+      case "expired":
+        return nfts?.filter(nft => nft.status !== "bonding");
+
+      default:
+        return nfts;
+    }
+  }, [bondFilter, nfts]);
+
   return (
-    <>
-      <h1>Lend</h1>
-      <Mint />
-      {data?.length && (
-        <ul>
-          {data
-            .sort((a, b) => Number(a.projection_id) - Number(b.projection_id))
-            .map(item => (
-              <li key={item.projection_id}>Chicken Bond #{item.projection_id}</li>
-            ))}
-        </ul>
-      )}
-    </>
+    <ClientOnly>
+      <div>
+        {!isConnected ? (
+          <div className="mx-auto mt-96 w-fit">
+            <Connect />
+          </div>
+        ) : (
+          <>
+            <AppHeader type="Lender" />
+            <main className="text-lg max-w-5xl w-4/5 mx-auto py-24">
+              <div className="flex justify-between mb-8">
+                <div className="flex gap-6">
+                  <button
+                    className={`btn-beige ${bondFilter === "none" ? "bg-offwhite" : ""}`}
+                    onClick={() => setBondFilter("none")}
+                  >
+                    All
+                  </button>
+                  <button
+                    className={`btn-beige ${bondFilter === "current" ? "bg-offwhite" : ""}`}
+                    onClick={() => setBondFilter("current")}
+                  >
+                    Current
+                  </button>
+                  <button
+                    className={`btn-beige ${bondFilter === "expired" ? "bg-offwhite" : ""}`}
+                    onClick={() => setBondFilter("expired")}
+                  >
+                    Expired
+                  </button>
+                </div>
+                <button className="btn-orange" onClick={() => console.log("mint")}>
+                  Mint new bond
+                </button>
+              </div>
+              {displayBonds?.length && (
```

Comment:
You have to explicitly say `displayBonds?.length > 0`, or else if length is 0 it will render the number 0. I have no idea why, but it happens 😂